opendatahub/manifests/base/kustomization.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../default
+  - ../prometheus
+  - ../overlays/odh
+
+commonLabels:
+  app.kubernetes.io/part-of: model-mesh
```

Comment:
Is this right to be here?
I am not able to find it on original manifests.

Reply:
looks like it was recently removed in [this commit](https://github.com/opendatahub-io/modelmesh-serving/commit/bdfdca83fce58703c42392e0cb8d7daecd5aa519). Removed it from this PR too.